packages/angular/pwa/pwa/files/assets/manifest.json

```diff
@@ -3,7 +3,7 @@
   "short_name": "<%= title %>",
   "theme_color": "#1976d2",
   "background_color": "#fafafa",
-  "display": "browser",
+  "display": "minimal-ui",
```

Comment:
Maybe a different default would be more appropriate. Some boilerplates use `standalone` (and this is also what we use in angular.io), but it depends on the app. `minimal-ui` seems to be the safest.